tests/baselines/reference/checkJsFiles_skipDiagnostics.symbols

```diff
@@ -0,0 +1,38 @@
+=== tests/cases/compiler/a.js ===
+
+var x = 0;
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+
+/// @ts-suppress
+x();
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+/// @ts-suppress
+x();
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+/// @ts-suppress
+x(
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+    2,
+    3);
+
+
+
+// @ts-suppress
+// come comment
+// some other comment
+
+// @anohter
+
+x();
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+
+
+// @ts-suppress: no call signature

```

Comment:
That is what i had in mind with test. but the error code seemed fairly useless. like you need to go somewhere to know what `TS 20124` means. Linter rules usually have descriptive names.

I think this is something we can target for in the future.